src/Common/src/CoreLib/System/Threading/Tasks/ValueTask.cs

```diff
@@ -33,9 +33,27 @@ namespace System.Threading.Tasks
     /// <summary>Provides an awaitable result of an asynchronous operation.</summary>
     /// <remarks>
     /// <see cref="ValueTask"/>s are meant to be directly awaited.  To do more complicated operations with them, a <see cref="Task"/>
-    /// should be extracted using <see cref="AsTask"/>.  Such operations might include caching an instance to be awaited later,
-    /// registering multiple continuations with a single operation, awaiting the same task multiple times, and using combinators over
-    /// multiple operations.
+    /// should be extracted using <see cref="AsTask"/>.  Such operations might include caching a task instance to be awaited later,
+    /// registering multiple continuations with a single task, awaiting the same task multiple times, and using combinators over
+    /// multiple operations:
+    /// <list type="bullet">
+    /// <item>
+    /// Once the result of a <see cref="ValueTask"/> instance has been retrieved, do not attempt to retrieve it again.
+    /// <see cref="ValueTask"/> instances may be backed by <see cref="IValueTaskSource"/> instances that are reusable, and such
```

Comment:
object vs instance guidance: When writing about an instance of a class, use object as a modifier on first mention in a major paragraph or section in a topic. You can omit the modifier thereafter if the context is clear. Use an article (a, an, the) before the name if it is singular. When referring to multiple instances of a class, do not form the plural by adding s or es to the class name; append the objects modifier instead.